pkg/skaffold/watch/files.go

```diff
@@ -71,7 +72,7 @@ func walk(dirs []string) (fileMap, error) {
 				return err
 			}
 
-			if isIgnored(info.Name()) {
+			if isIgnored(info.Name(), info.IsDir()) {
```

Comment:
I think the smallest change is to run those 3 liens only if `info.IsDir` is true. Th rest of the code doesn't have to change really.

Reply:
I fully agree that the minimal change is to change `isIgnored(info.Name(), info.IsDir())` by `info.isDir() && isIgnored(info.Name())`. The only problem I have with that, is that the `ignored` is restricted to directories and this become implicit. So there is a need to also change the variable name to `ignoredDir` imho.

Reply:
@loispostula very good idea indeed. Let's do that!